export const authLogin = async () => {
    try {
        
    } catch (err: any) {

    }
}